templates/javascript-ethers/scripts/whitelist-reader.js

```diff
@@ -0,0 +1,62 @@
+const fs = require("fs");
+const hre = require("hardhat");
+const { getServiceData, whitelistBeaconReader } = require("@api3/services");
+
+async function main() {
+  const network = hre.network.name;
+
+  // TODO: Update the mocked contract so that whitelisting is necessary
+  if (
+    network.toLowerCase() === "hardhat" ||
+    network.toLowerCase() === "localhost"
+  )
+    return;
+
+  // Read BeaconReaderExample contract address deployed using deploy.js
+  let beaconReaderExampleAddress = null;
+  try {
+    // TODO: Simplify
+    ({ beaconReaderExampleAddress } = JSON.parse(
+      fs.readFileSync(`./deployments/${network}.json`).toString()
+    ));
+    if (!beaconReaderExampleAddress) throw new Error("beaconId not found");
+  } catch (e) {
+    console.log(`Error: ${e}. Please try first running deploy script`);
+    return;
+  }
+
+  const deployments = getServiceData(
+    "Amberdata",
+    "global vwap for eth_usd",
+    network
+  );
+  beaconId = deployments.beacon.beaconId;
```

Comment:
beaconId seems to be missing let/const, isn't it?

Reply:
Good catch. Not sure if implicit `var` is used or it simply doesn't work at it's current state... I'll fix this in a following PR though.